cordova-react/www/scripts/bundle.js

```diff
@@ -0,0 +1,115 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
do not think this should be here. probably should be added to .gitignore as well.